app/controllers/stripe_active_cards_controller.rb

```diff
@@ -37,6 +38,7 @@ def update
     end
   rescue Stripe::CardError => e
     DataDogStatsClient.increment("stripe.errors")
+    DataDogStatsClient.increment("stripe.errors.update_subscription")
```

Comment:
The original error from Timber is searching the logs for _"Stripe Update Subscription Failure"_. I searched as far back as possible to see where exactly that error would be raised and I didn't it at all. My best guess was it happens here.